packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -89,6 +89,7 @@ export default function useTabNav() {
 
 			if ( event.keyCode === ESCAPE && ! hasMultiSelection() ) {
 				event.preventDefault();
+				event.stopPropagation();
```

Comment:
Is this still necessary?

Reply:
It's the same as https://github.com/WordPress/gutenberg/pull/54080#discussion_r1314962013 (ESCAPE).